src/UsgsAstroLsSensorModel.cpp

```diff
@@ -609,6 +615,22 @@ void UsgsAstroLsSensorModel::updateState()
    MESSAGE_LOG(m_logger, "updateState: half time duration set to {}",
                                m_halfTime)
 
+   // Compute if downtrack is increasing or decreasing lines
+   // This requires that all of the other parameters be set first!
+   csm::ImageCoord nextLinePoint((lineCtr+m_nLines)/2, sampCtr);

```

Comment:
I would also mention why we are even worried about the line track. Potentially mention the LROC and KaguyaTC problems we were/are having.